gists/frontend-pull-request-guidelines.md

```diff
@@ -124,5 +143,35 @@ This document details guidelines for reviewing frontend (especially LPL React-fl
   - Reevaluate your decision to create a separate component or look for other ways to compose the functionality together (e.g., use [`children`](https://react.dev/learn/passing-props-to-a-component#passing-jsx-as-children), move state down, use the [render prop pattern](https://reactjs.org/docs/render-props.html))
 - If you're referencing a hardcoded string (or enumeration of strings), place them in `config.js` and reference them via the aliased `config` import location
   - By convention, constants are cased using SCREAMING_SNAKE_CASE
+  - In general, double check your casing to confirm each is used appropriately
 - Never mutate a value held in the Redux store
   - https://redux.js.org/faq/immutable-data#why-is-immutability-required-by-redux
+- Keep your code clean. Remove empty brackets (or <React.Fragment>) if you make changes and there is now only one direct child element
+- Think about using "Provider" type components. We typically use these to make sure any child views have the data they need by wrapping those child views with the provider. The provider handles fetching data and storing it in redux. It can also make sure no child views are rendered until that data is available
```

Comment:
This is different from the "context" providers. Is that clear, or do you think I should add a clarifier?

Reply:
It is not clear 😅 . It's also pretty subjective and doesn't really align with the RSC paradigm. Back in the old days we called these Views (I think that's how our subsection generator still calls them). But I'm less bullish on this advice now based on where the framework is headed.